contracts/Comet.sol

```diff
@@ -201,6 +201,23 @@ contract Comet is CometMainInterface {
         (asset14_a, asset14_b) = getPackedAssetInternal(config.assetConfigs, 14);
     }
 
+    modifier nonReentrant() {
```

Comment:
I think one way to reduce contract size is to split these logic into two separate internal functions: `nonReentrantBefore` and `nonReentrantAfter`. It should save contract size because it uses a JUMP instruction instead of adding the bytecode to each function that uses the modifier.

Reply:
Sorry, just realized we went back and forth on this since you had it in separate functions originally 😅

Reply:
No problem :) 
The contract size reduced about 0.225 KiB 
(24.490KiB -> 24.266KiB)